### PR TITLE
ecos: 2.0.6 -> 2.0.7

### PR DESCRIPTION
**Pull request by drewrisinger**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by drewrisinger** [comment]
Depends on #79187

File: pkgs/development/libraries/science/math/ecos/default.nix
Comment (bhipple) [inline pkgs/development/libraries/science/math/ecos/default.nix]: I'll defer to someone like @jonringer, but is `downloadPage` an attribute we commonly put in `meta`?  Seems like it can be easily found from just looking at where `src` is downloading from.

**Review by bhipple — APPROVED** [review]

File: pkgs/development/libraries/science/math/ecos/default.nix
Comment (jonringer) [reply pkgs/development/libraries/science/math/ecos/default.nix]: commonly? no, but it is a support attribute, however, for github, you usually link to the releases page as that's where you would go to download a given version
```suggestion
    downloadPage = "https://github.com/embotech/ecos/releases";
```

Comment (drewrisinger) [reply pkgs/development/libraries/science/math/ecos/default.nix]: I thought it was somewhat useful as an easy link for reviewers/committers for getting the project, though it doesn't really contain more information than ``fetchFromGitHub``. Resolving @jonringer 's comment.

**Review by bhipple — APPROVED** [review]

**Review by jonringer — APPROVED** [review]
diff LGTM

```
[3 built, 0.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/79187
1 package built:
ecos
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build ecos

**Pull request merged**
